packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -259,6 +259,13 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
       $('#first').click();
       Terra.validates.element('first button clicked');
     });
+
+    it('Focus to single item on arrow up and arrow down', () => {
+      $('[class*=dropdown-button]').click();
+      $('[class*=dropdown-list]').waitForDisplayed();
+      browser.keys(['ArrowDown', 'ArrowDown', 'Enter']);
```

Comment:
```suggestion
      browser.keys(['ArrowDown', 'ArrowDown',]);
```
I think the test was added to ensure multiple ArrowDown was not thrown any error when list has just one item !! I

Reply:
Fixed.